.travis.yml

```diff
@@ -1,6 +1,14 @@
 language: node_js
-sudo: false
+services:
```

Comment:
We've been accustomed to setting credentials and other sensitive values through the Travis UI. We've been under the impression that that's more secure because then credentials don't appear here on github, even in an obfuscated for. I'll look into it a bit though and see if adding them here to `.travis.yml` is safer than we presumed. If so, that will be preferable, for sure. I'll get back to you on that.